MapboxNavigation/String.swift

```diff
@@ -25,7 +25,9 @@ extension String {
             ("&", "&amp;"),
             ("<", "&lt;"),
             ("\"", "&quot;"),
-            ("'", "&apos;")
+            ("'", "&apos;"),
+            ("(", ""),
+            (")", "")
```

Comment:
This turns `addingXMLEscapes` into a misnomer. This method may be used in places where we just want to mark up XML and don’t want to remove parentheses. Instead of repurposing this method, let’s modify the call sites to remove parentheses.